test/dlc_tests/eks/tensorflow/training/test_eks_tensorflow_multi_node_training.py

```diff
@@ -10,11 +10,14 @@
 from invoke import run
 import test.test_utils.ec2 as ec2_utils
 import test.test_utils.eks as eks_utils
-from test.test_utils import is_pr_context, SKIP_PR_REASON
+from test.test_utils import is_pr_context, SKIP_PR_REASON, is_tf1
 
 
 # Test only runs in region us-west-2, on instance type p3.16xlarge, on PR_EKS_CLUSTER_NAME_TEMPLATE cluster
 @pytest.mark.skipif(is_pr_context(), reason=SKIP_PR_REASON)
+# EKS multinode are failing on TF1 Pipeline due to scheduling issues.
+# TODO: Remove this line and add the required scheduling scheme.
+@pytest.mark.skipif(is_tf1(), reason="Skipping it on TF1 currently as it is not able to do the pods scheduling properly")
```

Comment:
doesn't is_tf1() require an image uri?

Reply:
check out https://github.com/aws/deep-learning-containers/pull/390/files for how to use is_tf1 in the way you're looking to

Reply:
Sorry, will make the change.